layouts/shortcodes/release-branches.html

```diff
@@ -0,0 +1,45 @@
+<h2 id="detailed-release-history-for-active-branches">Detailed Release History for Active Branches</h2>
+{{ range $releaseInfo := .Site.Data.releases.schedule.schedules }}
+{{- $version := printf "%.2f" $releaseInfo.release -}}
+{{- $dataIdVersion := replace ( printf "%.2f" $releaseInfo.release ) "." "-" | anchorize -}}
+
+<h3 id="{{ $dataIdVersion }}">{{ $version }}</h3>
+<p>
+    Next patch release is <strong>{{ $releaseInfo.next.release }}</strong>. {{ $releaseInfo.next.note }}
+</p>
+<p>
+    <strong>{{ $version }}</strong> enters maintenance mode on <strong><time datetime="{{ time.Format "2006-01-02" $releaseInfo.endOfLifeDate }}">{{ time.Format ( T "release_date_format") $releaseInfo.maintenanceModeStartDate }}</time></strong> and End of Life is on <strong><time datetime="{{ time.Format "2006-01-02" $releaseInfo.endOfLifeDate }}">{{ time.Format ( T "release_date_format") $releaseInfo.endOfLifeDate }}</time></strong>
```

Comment:
Nice use of the localized `release_date_format`.

Reply:
To make this work, you can load a _format string_ using the `T` localization function, and then format (eg) `<strong><time datetime="{{ time.Format "2006-01-02" $releaseInfo.endOfLifeDate }}">{{ time.Format ( T "release_date_format") $releaseInfo.maintenanceModeStartDate }}</time></strong>` into that string (use `safeHTML` if you need to).

Document to the localization teams that the string you define is a format string.